build.gradle

```diff
@@ -139,7 +139,7 @@ codenarc {
     toolVersion = '1.6'
     configFile = file('codenarc.groovy')
     maxPriority1Violations = 0
-    maxPriority2Violations = 1
+    maxPriority2Violations = 2
```

Comment:
There method causing the codenarc violation is the one for creating an issue of type Security Vulnerability which now has 6 parameters (I added fixVersion and components) and I think keeping the method is the cleanest way.

Reply:
Replaced with named arguments.